src/index.ts

```diff
@@ -137,6 +148,32 @@ export class WaxJS {
     return this.user.account;
   }
 
+  private async getChainInfoByChainName(chainName: string): Promise<any> {
+    const response = await this.rpc.get_table_rows({
+      json: true,
+      code: "registry.wax",
+      scope: "registry.wax",
+      table: "chains",
+    });
+    const chainInfo: any | null = response.rows.find(row => row.chain_name === chainName);
+
+    if (!chainInfo) {
+      // TODO: handle case chain info not exists
```

Comment:
I think it is ok to return null or undefined if it doesn't exist. Other places in the code should throw an error if the logic needs to find a valid chain